### PR TITLE
demo(input): add input a11y examples

### PR DESCRIPTION
**Pull request by mmalerba**

### CODE REVIEW

File: src/demo-app/a11y/input/input-a11y.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<section>
+  <h2>Basic input box (e.g. name field)</h2>
+  <md-input-container floatPlaceholder="never">
+    <input mdInput placeholder="First name" [(ngModel)]="firstName">
```
Comment (tinayuangao) [inline src/demo-app/a11y/input/input-a11y.html]: add `aria-label`?

File: src/demo-app/a11y/input/input-a11y.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  </md-input-container>
+  =
+  <md-input-container floatPlaceholder="always">
+    <input mdInput type="number" placeholder="JPY" [(ngModel)]="jpy">
+    <span mdPrefix>‎¥‎</span>
+  </md-input-container>
+  (as of 7/31/2017)
+</section>
+
+<section>
```
Comment (tinayuangao) [inline src/demo-app/a11y/input/input-a11y.html]: add header for this section?

File: src/demo-app/a11y/input/input-a11y.html (hunk 1 above)
Comment (mmalerba) [inline src/demo-app/a11y/input/input-a11y.html]: Not needed when you specify a placeholder since it gets placed inside of a `<label>` tag

**Comment by mmalerba** [comment]
rebased and addressed comments

**Pull request merged**